### DISCUSSION TITLE
Java classes generated from XSD schemas

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#520] [Q&A] [answered]
Has anyone generated java classes (POJOs) from the SDK schemas (xsd)? If so, could you share it?

**Comment by rouschr** [comment, marked as answer]
We do not do this in the editor demo as we dynamically build the XML based on fields.json metadata.
But you could have a look at various ways to do it here:
https://stackoverflow.com/questions/11463231/how-to-generate-jaxb-classes-from-xsd
